scripts/docker-as-artifacts

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+set -e
+
+: ${ARTIFACTS_INSTALL:="/usr/bin/artifacts-install"}
+: ${ARTIFACTS_DEFAULT_DIR:="/var/www/log"}
+: ${ARTIFACTS_DEFAULT_S3:="${CI_REPO_NAME}/${CI_BUILD_NUMBER}"}
+
+# Artifacts variables
+: ${ARTIFACTS_BIN:="/usr/bin/artifacts"}
+export ARTIFACTS_KEY=${ARTIFACTS_KEY:=""}
+export ARTIFACTS_SECRET=${ARTIFACTS_SECRET:=""}
+export ARTIFACTS_BUCKET=${ARTIFACTS_BUCKET:="codeship-artifact"}
+export ARTIFACTS_REGION=${ARTIFACTS_REGION:="eu-west-1"}
+
```

Comment:
```suggestion
if [ -z "$ARTIFACTS_KEY" ] || [ ! -z "$ARTIFACTS_SECRET" ]; then
  echo "Error: You need to define env variables ARTIFACTS_KEY and ARTIFACTS_SECRET with S3 credentials"
  exit 1
fi

```

Reply:
We could use color as we did on `docker-as-drupal` ?

```
printf "\e[01;31mError: You need to define env variables ARTIFACTS_KEY and ARTIFACTS_SECRET with S3 credentials.\e[0m\n"
```

Reply:
True about `[ ! -z "$ARTIFACTS_SECRET" ]`